src/modm/platform/extint/rp/module.lb

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2022, Nikolay Semenov
+#
+# This file is part of the modm project.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# -----------------------------------------------------------------------------
+
+def init(module):
+    module.name = ":platform:irq"
+    module.description = FileReader("module.md")
+
+def prepare(module, options):
+    module.depends(
+        ":cmsis:device",
+        ":platform:gpio")
+    module.add_option(
+        BooleanOption(
+            name="gpio",
+            description="Enable IRQ support for GPIO",
+            default=True))
+    module.add_option(
+        BooleanOption(
+            name="qspi",
+            description="Enable IRQ support for QSPI",
+            default=False))
+    module.add_option(
+        StringOption(
+            name="handler.storage",
+            description="Size of storage for the handler's implementation",
+            default="sizeof(void*)"))
```

Comment:
Let's make this a CPP define, since that is the convention in the rest of the library.
We want to utilize as much of the native toolchain features and only use lbuild for the rest.